src/Form/TypeGuesser/CronExpressionTypeGuesser.php

```diff
@@ -34,7 +35,6 @@ public function guessType($class, $property): ?TypeGuess
             return null;
         }
 
-
         $types = $this->extractor->getTypes($class, $property);
         if (!$types) {
```

Comment:
@Chris53897 before I make an extra PR, use `if (is_null($types)) {` to fix RiskyTruthyFalsyComparison

Reply:
What I hate there, is that the code-style fixers can't keep straight if it should be `is_null` or `null ===` :(